dts/bindings/dai/nxp,dai-sai.yaml

```diff
@@ -12,100 +12,46 @@ properties:
     required: true
   mclk-is-output:
     type: boolean
-    description: |
-      Use this property to set the SAI MCLK as output or as input.
-      By default, if this property is not specified, MCLK will be
-      set as input. Setting the MCLK as output for SAIs which don't
-      support MCLK configuration will result in a BUILD_ASSERT()
-      failure.
+    description: MCLK is configured as output.
   rx-fifo-watermark:
     type: int
-    description: |
-      Use this property to specify the watermark value for the TX
-      FIFO. This value needs to be in FIFO words (NOT BYTES). This
-      value needs to be in the following interval: (0, DEFAULT_FIFO_DEPTH],
-      otherwise a BUILD_ASSERT() failure will be raised.
+    description: Watermark value (in FIFO words) of RX FIFO.
   tx-fifo-watermark:
     type: int
-    description: |
-      Use this property to specify the watermark value for the RX
-      FIFO. This value needs to be in FIFO words (NOT BYTES). This
-      value needs to be in the following interval: (0, DEFAULT_FIFO_DEPTH],
-      otherwise a BUILD_ASSERT() failure will be raised.
+    description: Watermark value (in FIFO words) of TX FIFO.
   interrupts:
     required: true
   fifo-depth:
     type: int
-    description: |
-      Use this property to set the FIFO depth that will be reported
-      to other applications calling dai_get_properties(). This value
-      should be in the following interval: (0, DEFAULT_FIFO_DEPTH],
-      otherwise a BUILD_ASSERT() failure will be raised.
-      By DEFAULT_FIFO_DEPTH we mean the actual (hardware) value of
-      the FIFO depth. This is needed because some applications (e.g: SOF)
-      use this value to compute the DMA burst size, in which case
-      DEFAULT_FIFO_DEPTH cannot be used. Generally, reporting a false
-      FIFO depth should be avoided. Please note that the sanity check
-      for tx/rx-fifo-watermark uses DEFAULT_FIFO_DEPTH instead of this
-      value so use with caution. If unsure, it's better to simply not
-      use this property, in which case the reported value will be
-      DEFAULT_FIFO_DEPTH.
+    description: Size (in FIFO words) of the TX/RX FIFO.
```

Comment:
I find some of the description about how to pick the value (between 0 to DEFAULT_FIFO_DEPTH useful to include here.